src/Google.Cloud.Functions.Testing/FunctionTestServer.cs

```diff
@@ -47,12 +48,12 @@ public FunctionTestServer(Type functionTarget) : this(functionTarget, null)
 
         /// <summary>
         /// Creates a FunctionTestServer for the specified function type, with an optional
-        /// type to inspect for <see cref="FunctionTestStartupAttribute"/> attributes.
+        /// type to inspect for <see cref="FunctionsStartupAttribute"/> attributes.
         /// </summary>
         /// <param name="functionTarget">The function type to host in the server.</param>
-        /// <param name="attributedStartupType">A type to examine for <see cref="FunctionTestStartupAttribute"/> attributes to provide further customization.
-        /// If this is null, or no attributes are specified in the type (or its base class hierarchy) the startup classes in the
-        /// function assembly are used.</param>
+        /// <param name="attributedStartupType">A type to examine for <see cref="FunctionsStartupAttribute"/> attributes to provide further customization.
```

Comment:
nit: The name of this param seems backwards to me. Maybe startupSourceType? typeWithStartups? maybe even startupAttributedType? As it is it just looks like we are passing the Startup type.